modules/core/src/main/java/io/swagger/assert4j/assertions/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Assertion builders and utilities
```

Comment:
End every phrase like this with a period. That's how Sun/Oracle did it in their Javadocs at least.